src/main/steps/tasklistresponse/summary/postController.ts

```diff
@@ -1,21 +1,20 @@
 import autobind from 'autobind-decorator';
+import config from 'config';
 import { Response } from 'express';
 
-import { CosApiClient } from '../../../app/case/CosApiClient';
-import { toApiFormat } from '../../../app/case/to-api-format';
 import { AppRequest } from '../../../app/controller/AppRequest';
 import { AnyObject, PostController } from '../../../app/controller/PostController';
 import { Form, FormFields, FormFieldsFn } from '../../../app/form/Form';
-import { CA_RESPONDENT_RESPONSE_CONFIRMATION } from '../../urls';
+import PCQGetController from '../../common/equality/get';
+import { RESPONDENT_TO_APPLICATION_SUMMARY_REDIRECT } from '../../urls';
 
 @autobind
-export default class ResponseSummaryConfirmationPostController extends PostController<AnyObject> {
+export default class ResponseSummaryConfirmationPcqPostController extends PostController<AnyObject> {
```

Comment:
This shd be ResponseSummaryConfirmationPostController. call PCQ launcher method from here